### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by reaction-justin**
Corrected a couple of small typos and infelicities of language.

Impact: **minor**  
Type: **chore**

## Issue
There were a few corrections I needed to make to the Readme text.

## Solution
Made the corrections.

### CODE REVIEW

**Comment by machikoyasuda** [comment]
@spencern Is it okay if we merge this into `master` without it going into `develop`? Or should he made a new PR into `develop`?

File: README.md [hunk 1]
```diff
@@ -81,4 +81,4 @@ If you forget to sign your commits, the DCO bot will remind you and give you det
 
 ### License
 
-Copyright © [GNU General Public License v3.0](./LICENSE.md)
\ No newline at end of file
+Copyright © [GNU General Public License v3.0](./LICENSE.md)
```
Comment (spencern) [inline README.md]: What's changed on this line?

Comment (aldeed) [reply README.md]: 🤷🏽‍♂️

**Comment by aldeed** [comment]
Anytime something is merged directly to master, be sure to immediately merge `master` into `develop` and push `develop`. Not sure if we require a PR for that step.

File: README.md (hunk 1 above)
Comment (machikoyasuda) [reply README.md]: Ryan's PR did this too: https://github.com/reactioncommerce/reaction/pull/5319/files#diff-04c6e90faac2675aa89e2176d2eec7d8L84 

Must be GitHub.com.

**Pull request closed without merging**